internal/npm_install/test/golden/jasmine/BUILD.bazel.golden

```diff
@@ -0,0 +1,10 @@
+
+package(default_visibility = ["//visibility:public"])
+alias(
+    name = "jasmine",
+    actual = "//node_modules/jasmine",
+)
+alias(
+    name = "jasmine__bin",
```

Comment:
should the README say something about how we create the `nodejs_binary` for any package that has a `bin` and what label you use to reference it?

Reply:
👍